modules/clearCompletedTask.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable import/no-cycle */
```

Comment:
- [ ] [OPTIONAL] Kindly consider fixing linters instead of ignoring the errors, linters can make your code clean